JSDemos/Demos/TreeList/CustomizeKeyboardNavigation/React/App.tsx

```diff
@@ -24,7 +24,7 @@ const App = () => {
   const [enterKeyDirection, setEnterKeyDirection] = React.useState<TreeListTypes.EnterKeyDirection>('column');
 
   return (
-    <div id="tree-list-demo">
+    <React.Fragment>
```

Comment:
Question.

Can we use the shortcut ```<>...</>``` instead of the ```<React.Fragment>...</React.Fragment>``` in our React demos?

Reply:
We can't, linter replaces it with `React.Fragment`. I don't know why